open_gpt/cli/commands/deploy.py

```diff
@@ -0,0 +1,121 @@
+from cleo.commands.command import Command
+from cleo.helpers import argument, option
+
+
+class DeployCommand(Command):
+    name = "deploy"
+
+    description = "Start to deploy a model to JCloud or AWS."
+
+    arguments = [argument("model_name", "The name of the model to serve.")]
+    options = [
+        option(
+            'grpc_port',
+            None,
+            'The gRPC port to serve the model on.',
+            flag=False,
+            default=51001,
+        ),
+        option(
+            'http_port',
+            None,
+            'The HTTP port to serve the model on.',
+            flag=False,
+            default=51002,
+        ),
+        option(
+            'name',
+            None,
+            'The name of the deployment.',
+            flag=False,
+            default='open-gpt-deployment',
+        ),
+        option(
+            'cloud',
+            None,
+            'The cloud to deploy the model on.',
+            flag=False,
+            default='jcloud',
```

Comment:
As mentioned in README, we use `jina` as the cloud name rather than `jcloud`. Please keep it consistent.